### PR TITLE
Create spec.md

### PR DESCRIPTION
**Pull request by safrazier17**
markdown version of the pre-public specification from the working group as of 25 mar 2022

### CODE REVIEW

**Comment by thekaveman** [comment]
Taking a look at this 👀

File: docs/spec.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+| deadheads.txt | Defines scheduled deadheads contained in a transit feed. (This file is analogous to trips.txt for non-revenue operations.) |
+| ops_locations.txt | Significant operational locations relevant to the performance of vehicle deadheads. (This file is analogous to stops.txt for non-revenue operations.) |
+| deadhead_times.txt | Times that a vehicle arrives at and departs from operational locations for each deadhead. (This file is analogous to stop_times.txt for non-revenue operations.) |
+
+### deadheads.txt
+
+| **Field Name** | **Type** | **Required** | **Description** |
+| --- | --- | --- | --- |
+| deadhead_id | ID | Required | Identifies a deadhead. |
+| service_id | ID referencing **calendar.service_id** | Required | Identifies a set of dates when the deadhead is scheduled to take place. |
```
Comment (thekaveman) [inline docs/spec.md:24]: Does it make sense to link to the GTFS spec/files where we reference them, such as here?

**Review by thekaveman — COMMENTED** [review]
I made a few minor cleanups of the heading levels.

I generally agree with @e-lo's comments RE: building out some of the content a bit more. My take is that we should figure out the one place to context-set (likely the homepage) and move that kind of content there, leaving the spec page to describe the file layout, relationship to GTFS, etc.

The spec tables are clear and easy to follow!

**Review by safrazier17 — COMMENTED** [review]
addressed comments by @e-lo and @thekaveman, also made minor text edits in the runs.txt file based on suggestions from Matthias Gruetzner

**Review by thekaveman — APPROVED** [review]
This is looking great to me 👍 I checked the preview site and clicked all the links, they work.

**Pull request merged**